tests/10-EdvClient.spec.js

```diff
@@ -6,9 +6,10 @@ import mock from './mock.js';
 import {isRecipient} from './test-utils.js';
 
 describe('EdvClient', () => {
-  let invocationSigner, keyResolver = null;
+  let invocationSigner, keyResolver, kAK = null;
   before(async () => {
     await mock.init();
+    kAK = mock.keys.keyAgreementKey;
```

Comment:
this is a similar situation to the acronym URL.  In these cases I think it's best to just make it all lower case.  So in this case `kak`.

Reply:
Updated 437f761